src/Aspire.Dashboard/README.md

```diff
@@ -17,13 +17,38 @@ For example, the default certificate used by HTTPS endpoints can be configured u
 
 The OTLP endpoint can be secured with [client certificate](https://learn.microsoft.com/aspnet/core/security/authentication/certauth) or API key authentication.
 
-- `DOTNET_DASHBOARD_OTLP_AUTH_MODE` specifies the authentication mode on the OTLP endpoint. Possible values are `Certificate`, `ApiKey`, `None`. This configuration is required.
-- `DOTNET_DASHBOARD_OTLP_API_KEY` specifies the API key for the OTLP endpoint when API key authentication is enabled. This configuration is required for API key authentication.
+- `Otlp:AuthMode` specifies the authentication mode on the OTLP endpoint. Possible values are `Certificate`, `ApiKey`, `Unsecured`. This configuration is required.
+- `Otlp:ApiKey` specifies the API key for the OTLP endpoint when API key authentication is enabled. This configuration is required for API key authentication.
 
 ## Resources
 
 - `DOTNET_RESOURCE_SERVICE_ENDPOINT_URL` specifies the gRPC endpoint to which the dashboard connects for its data. There's no default. If this variable is unspecified, the dashboard shows OTEL data but no resource list or console logs.
 
+The resource service client supports certificates. Set `ResourceServiceClient:AuthMode` to `Certificate`, then add the following configuration:
+
+- `ResourceServiceClient:ClientCertificate:Source` (required) one of:
+  - `File` to load the cert from a file path, configured with:
+    - `ResourceServiceClient:ClientCertificate:FilePath` (required, string)
+    - `ResourceServiceClient:ClientCertificate:Password` (optional, string)
+  - `KeyStore` to load the cert from a key store, configured with:
+    - `ResourceServiceClient:ClientCertificate:Subject` (required, string)
+    - `ResourceServiceClient:ClientCertificate:KeyStore:Name` (optional, [`StoreName`](https://learn.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.storename), defaults to `My`)
+    - `ResourceServiceClient:ClientCertificate:KeyStore:Location` (optional, [`StoreLocation`](https://learn.microsoft.com/dotnet/api/system.security.cryptography.x509certificates.storelocation), defaults to `CurrentUser`)
+- `ResourceServiceClient:Ssl` (optional, [`SslClientAuthenticationOptions`](https://learn.microsoft.com/dotnet/api/system.net.security.sslclientauthenticationoptions))
+
+To opt-out of authentication, set `ResourceServiceClient:AuthMode` to `Unsecured`. This completely disables all security for the resource service client. This setting is used during local development, but is not recommended if you attempt to host the dashboard in other settings.
+
+## Dashboard web app auth
+
+The dashboard's web application supports OpenID Connect. Set `DashboardWebApp:AuthMode` to `OpenIdConnect`, then add the following configuration:
+
+- `Authentication:Schemes:OpenIdConnect:Authority` &mdash; URL to the identity provider (IdP)
```

Comment:
This is fine for now but I'd like to see if we have move this auth configuration to `Dashboard:Frontend:OpenIdConnect`. I think we can do that by binding another config section.

Can happen in a future PR.

Reply:
`Authentication:Schemes:<schemename>` seems to be the de facto way of doing this, no? If we move to our own config container, I'd be concerned we miss out on other built-in configuration controls.

Reply:
Schemes include functionality to bind their options to a convention-based location in `IConfiguration`. It's 1 line of code to add support for binding to an extra location.

I don't know yet whether we want to do this. If there are sacrifices, then it might be left at the default location.